plugins/org.yakindu.sct.doc.user/src/user-guide/code_generation_c.textile

```diff
@@ -14,19 +14,16 @@ h4(#codegen_c_identifiersettings_feature). IdentifierSettings feature
 The *IdentifierSettings* feature allows the configuration of module names and identifier character length:
 
 * _moduleName_ (String, optional): Name for header and implementation. Default: statechart name.
-* _statemachinePrefix_ (Boolean, optional): Prefix that is prepended to function, state, and type names.
```

Comment:
Why did we remove the prefix?

Reply:
Because it affects the generated name and, because it lives in the sgen, is not visible from the statechart and its validation. Also you could just rename the statechart to have the same effect...?